testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/clients/ClientClientScopesTest.java

```diff
@@ -35,6 +34,8 @@
 import org.keycloak.testsuite.console.page.clients.clientscopes.ClientScopesSetupForm;
 import org.keycloak.util.JsonSerialization;
 import org.keycloak.util.TokenUtil;
+import org.hamcrest.MatcherAssert;
+import static org.hamcrest.core.IsCollectionContaining.hasItems;
```

Comment:
Prefer import from `Matchers` rather than individual hamcrest classes that can change in the future:
```suggestion
import static org.hamcrest.Matchers.hasItems;
```